react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -118,7 +118,7 @@ public void onReactContextInitialized(ReactContext context) {
 
         // The string here (e.g. "MyReactNativeApp") has to match
         // the string in AppRegistry.registerComponent() in index.js
-        mReactRootView.startReactApplication(mReactInstanceManager, "gutenberg", initialProps);
+        mReactRootView.setAppProperties(initialProps);
```

Comment:
I don't know a lot about the bridge but this seems odd. If the app hasn't started yet, why are we setting the properties?

Reply:
The properties are kept even though the App may not have been started yet. Looking at the RN source seems to indicate that this is fine.

I can do some more tests in the meantime.

Reply:
Did some runtime debugging and indeed, `setAppProperties()` won't try to run the app since this condition fails: https://github.com/facebook/react-native/blob/0.57-stable/ReactAndroid/src/main/java/com/facebook/react/ReactRootView.java#L463

Reply:
I was thinking less about it being problematic than it being useless, since we'll just set those on initContent, right?

Reply:
Oh, not useless. It passes the `htmlModeEnabled` flag that is currently relayed via `onCreateView`. It is then preserved in `initContent()` because of the `getAppProperties()` call there. I could try to refactor this all but I preferred to keep the PR to the minimum of change possible.

Reply:
I see, that's the bit of context I was missing 👍